templates/certificates/_accomplishment-rendering.html

```diff
@@ -3,110 +3,217 @@
 
-                    % if organization_logo:
-                    <li class="wrapper-organization">
-                        <div class="organization">
-                            <img class="organization-logo" src="${organization_logo.url}" alt="${platform_name}">
-                        </div>
-                    </li>
-                    % endif
-                </ul>
-            </div>
-        </div>
+<div class="wrapper-accomplishment-rendering">
+  <div class="accomplishment-rendering">
+    <div class="ar-header">
+      <h1>${platform_name}<br/>Honor Certificate of Completion</h1>
+      <div class="ar-header-logo">
+	<img src="/static/images/ms-logo-white.png" alt="${platform_name}" />
+      </div>
     </div>
-
-    <div class="wrapper-accomplishment-metadata">
-        <div class="accomplishment-metadata">
-            <h2 class="accomplishment-metadata-title hd-6">${accomplishment_copy_more_about | h}</h2>
-
-            <div class="wrapper-metadata">
-                <dl class="metadata accomplishment-recipient">
-                    <dt class="label sr-only">Awarded to:</dt>
-                    <dd class="value copy copy-meta">
-                        <span class="recipient-img">
-                            <img class="src" src="/static/certificates/images/demo-user-profile.png" alt="">
-                        </span>
-                        <div class="recipient-details">
-                            <h3 class="recipient-name">${accomplishment_copy_name | h}</h3>
-                            <p class="recipient-username">${accomplishment_copy_username} @ ${platform_name}</p>
-                        </div>
-                    </dd>
-                </dl>
-                <dl class="metadata accomplishment-id">
-                    <dt class="label copy copy-meta">${certificate_id_number_title}:</dt>
-                    <dd class="value copy copy-base">${certificate_id_number}</dd>
-                </dl>
-                <dl class="metadata accomplishment-date">
-                    <dt class="label copy copy-meta">${certificate_date_issued_title}</dt>
-                    <dd class="value copy copy-base">${certificate_date_issued}</dd>
-                </dl>
-            </div>
-        </div>
+   <div class="ar-ribbon">			
+   </div> 
+   <div class="ar-body">
+     <div class="ar-body-content">
+       <h2>${accomplishment_copy_name | h}</h2>
+       <p>${accomplishment_copy_description_full}</p>
+       <h3>
+          <span class="accomplishment-course-org">${accomplishment_copy_course_org}  </span>
+          <span class="accomplishment-course-number">${course_number}</span>:
+          <span class="accomplishment-course-name">${accomplishment_copy_course_name}</span>
+       </h3>
+       <p>${accomplishment_copy_course_description}</p>
+       <div class="ar-body-content-signs">
+         <div class="ar-signature">
+	   <img src="/static/images/satya-sign.png"/>
```

Comment:
nit: missing alt tag. not going to block on this, because we'll eventually clean this up during accessibility sweeps